lib/active_model_serializers/rspec/matchers/serializer.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_model_serializers/test/serializer'
+
+module ActiveModelSerializers
+  module RSpecMatchers
+    module Serializer
+      extend ActiveModelSerializers::Test::Serializer
+      extend ActiveSupport::Concern
+
+      included do
+        setup :setup_serialization_subscriptions
+        teardown :teardown_serialization_subscriptions
+
+        RSpec::Matchers.define :use_serializer do |expected|
```

Comment:
I'd rather just not include the `use_serializer` in this PR if even @leonelgalan doesn't use it.